src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
```

Comment:
in delComment you might want to add error handling in case the server returns a non-2xx status code. You could throw an error or return a custom object with an error message.
example:
```
const result = await fetch(`${API_BASE_URL}/comments/${id}`, { method: 'DELETE' });

  if (!result.ok) {
    throw new Error(`${result.status} - ${result.statusText}`);
  }

  return result;
```